README.md

```diff
@@ -144,9 +143,29 @@ framework:
         transports:
             async: 
                 dsn: 'https://sqs.us-east-1.amazonaws.com/123456789/my-queue.fifo'
-                options: { message_group_id: com_example }
```

Comment:
This could be considered a BC break

Reply:
It is fine to break BC if we update to 0.4.0